source/math/FloatSize2D.ooc

```diff
@@ -25,7 +25,7 @@ FloatSize2D: cover {
 	length ::= this norm
 	empty ::= !(this width > 0 && this height > 0)
 	norm ::= (this width squared() + this height squared()) sqrt()
-	azimuth ::= this height atan2(this width)
+	azimuth ::= atan2(this height, this width) as Float
```

Comment:
That's true, but it makes `atan2` a global function of sorts, and we're trying to avoid that, judging from the rest of the code (`this width abs()` and so on).